titration/utils/devices/liquid_crystal.py

```diff
@@ -1,22 +1,46 @@
+"""
+The file for the Sunfire LCD 20x04 Char Display, LiquidCrystal Class
+"""
 import time
-
 import digitalio
+import board
+
+# LCD Device Constants
+LCD_CHR = True
```

Comment:
Why global variables?

Reply:
Rather than keeping all constants in one file, I thought it would be better that if the constants are only used in one file, that they should be placed in that file.